java-spring6/src/main/java/nz/co/blink/debit/client/v1/EnduringConsentsApiClient.java

```diff
@@ -121,90 +120,90 @@ public Mono<CreateConsentResponse> createEnduringConsent(EnduringConsentRequest
     public Mono<CreateConsentResponse> createEnduringConsent(EnduringConsentRequest request, final String requestId)
             throws BlinkServiceException {
         if (request == null) {
-            throw new BlinkInvalidValueException("Enduring consent request must not be null");
```

Comment:
why not keep the invalid value exception? i.e. as a subclass of the Service Exception